pkg/util/optional/uint.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package optional
+
+import (
+	"fmt"
+	"math"
+)
+
+// MakeUint returns an Uint with a set value. The maximum value that can be
+// represented is (MaxUint64-1).
+func MakeUint(value uint64) Uint {
+	if value == math.MaxUint64 {
+		return Uint{
```

Comment:
Drive-by comment: should we have made this case panic? Is there a benefit to silently accepting a value that won't round-trip through the optional wrapper?

Reply:
I debated that too. Since currently this is used for stats, I went this way to avoid a potentially unhandled panic.